### PR TITLE
do not install pip deps for some integrations

### PR DESCRIPTION
**Pull request by gmmeyer**
This should fix the integrations issue. It is not a great solution: I would vastly prefer to be able to do ship them with upgraded dependencies. However, this is the solution we have. So, as sad as I am at the moment to give up the dream of shipping updated deps, it's all we can do. 😢

### CODE REVIEW

File: resources/datadog-integrations/software.rb.erb [hunk 1]
```diff
@@ -52,27 +52,28 @@ build do
...
-
   if File.exists? "#{integration_source_folder}/requirements.txt"
-    # install requirements
-    install_cmd = "install --target=#{integration_agent_folder}/lib "
-    # Only apply the constraints file if one exists
-    if File.exists? "#{install_dir}/agent/requirements.txt"
-      install_cmd += " -c #{install_dir}/agent/requirements.txt "
-    end
-    install_cmd += " -r #{integration_source_folder}/requirements.txt"
+    unless manifest.key?("package_deps") && ! manifest["package_deps"]
```
Comment (truthbk) [inline resources/datadog-integrations/software.rb.erb]: Wouldn't this be easier to read with an `if` instead?

File: Rakefile.rb [hunk 2]
```diff
@@ -71,11 +71,18 @@ def windows?()
   desc 'Build all integrations'
   task :'build-all-integrations' do
     checks = Dir.glob("/#{ENV['INTEGRATIONS_REPO']}/*/")
+    if ENV['SKIP_INTEGRATION']
+      skip_checks = ENV['SKIP_INTEGRATION'].split(',')
```
Comment (truthbk) [inline Rakefile.rb]: would it be better to split and strip?

```
skip_checks = ENV['SKIP_INTEGRATION'].split(",").map(&:strip)
```

File: resources/datadog-integrations/software.rb.erb (hunk 1 above)
Comment (gmmeyer) [inline resources/datadog-integrations/software.rb.erb]: I can change it to an if if you'd like!

Comment (gmmeyer) [inline resources/datadog-integrations/software.rb.erb]: I agree that it's clearer, the original was definitely unclear

File: Rakefile.rb (hunk 2 above)
Comment (gmmeyer) [inline Rakefile.rb]: Yes, thats a good idea

**Pull request merged**